__tests__/types.test.js

```diff
@@ -14,6 +21,36 @@ describe('validator module', () => {
       expect(isNumber(() => {})).toBeFalsy();
       expect(isNumber(true)).toBeFalsy();
     });
+    it('properly tells if a value is a string', () => {
+      expect(isString('hi')).toBeTruthy();
+      expect(isString('hello, my name is willow')).toBeTruthy();
+      expect(isString('5')).toBeTruthy();
+      expect(isString('false')).toBeTruthy();
+      expect(isString(5)).toBeFalsy();
+      expect(isString(true)).toBeFalsy();
+      expect(isString([])).toBeFalsy();
+    });
+    it('properly tells if a value is a boolean', () => {
+      expect(isBoolean(true)).toBeTruthy();
+      expect(isBoolean(false)).toBeTruthy();
+      expect(isBoolean('hi')).toBeFalsy();
+      expect(isBoolean(5)).toBeFalsy();
+      expect(isBoolean('false')).toBeFalsy();
+    });
+    it('properly tells if a value is an array', () => {
+      expect(isArray([1])).toBeTruthy();
+      expect(isArray([2, 4, 7])).toBeTruthy();
+      expect(isArray(['cat', 'dog', 'bird'])).toBeTruthy();
+      expect(isArray(6)).toBeFalsy();
+      expect(isArray('yes')).toBeFalsy();
+    });
+    it('properly tells if a value is an object', () => {
+      expect(isObject({})).toBeTruthy();
+      expect(isObject({ 'type': 'cat' })).toBeTruthy();
+      expect(isObject({ 'type': 'cat', 'name': 'willow' })).toBeTruthy();
+      expect(isObject(5)).toBeFalsy();
+      expect(isObject('cat')).toBeFalsy();
+    });
```

Comment:
you should add a test case for checking arays. 
`      expect(isObject([])).toBeFalsy();
`
this would fail though. you'd have to work on our isObject function